<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TemplatePreviewToolView</class>
 <widget class="QWidget" name="TemplatePreviewToolView">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>686</width>
    <height>571</height>
   </rect>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="KMessageWidget" name="messageWidget" native="true"/>
   </item>
   <item>
    <widget class="TemplatePreview" name="preview" native="true"/>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox_2">
     <property name="title">
      <string>Template type</string>
     </property>
     <property name="flat">
      <bool>true</bool>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <widget class="QRadioButton" name="projectRadioButton">
        <property name="text">
         <string>Project template</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="classRadioButton">
        <property name="text">
         <string>Class template</string>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <spacer name="horizontalSpacer">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeType">
           <enum>QSizePolicy::Maximum</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QLabel" name="emptyLinesPolicyLabel">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="text">
           <string>&amp;Empty lines policy</string>
          </property>
          <property name="buddy">
           <cstring>emptyLinesPolicyComboBox</cstring>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="emptyLinesPolicyComboBox">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <item>
           <property name="text">
            <string>Keep empty lines</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Remove repeated empty lines</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Remove all empty lines</string>
           </property>
          </item>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer_2">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>KMessageWidget</class>
   <extends>QWidget</extends>
   <header>kmessagewidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>TemplatePreview</class>
   <extends>QWidget</extends>
   <header>templatepreview.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>classRadioButton</sender>
   <signal>toggled(bool)</signal>
   <receiver>emptyLinesPolicyComboBox</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>76</x>
     <y>69</y>
    </hint>
    <hint type="destinationlabel">
     <x>171</x>
     <y>100</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>classRadioButton</sender>
   <signal>toggled(bool)</signal>
   <receiver>emptyLinesPolicyLabel</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>55</x>
     <y>70</y>
    </hint>
    <hint type="destinationlabel">
     <x>55</x>
     <y>101</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
